lib/obfuscate_id.rb

```diff
@@ -4,8 +4,9 @@ def obfuscate_id(options = {})
 
     extend ClassMethods
     include InstanceMethods
-    cattr_accessor :obfuscate_id_spin
+    cattr_accessor :obfuscate_id_spin, :obfuscater # Obfuscater, aka the ScatterSwapper for this spin
     self.obfuscate_id_spin = (options[:spin] || obfuscate_id_default_spin)
+    self.obfuscater = ScatterSwap::Hasher.new(self.obfuscate_id_spin)
   end
 
   def self.hide(id, spin)
```

Comment:
These `hide` and `show` methods need to be updated to the new method signatures on the ScatterSwap methods. Looks like only one param required.